Chinese/推荐美剧/stranger.things.md

```diff
@@ -0,0 +1,3 @@
+In a small town where everyone knows everyone| a peculiar incident starts a chain of events that leads to the disappearance of a child - which begins to tear at the
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.